test/stat/stat_test.go

```diff
@@ -88,6 +88,21 @@ func TestCliStatForConduitNamespace(t *testing.T) {
 			t.Fatal(err.Error())
 		}
 	})
+
+	t.Run("test named deploy query", func(t *testing.T) {
```

Comment:
I'm not sure this needs an integration test, since it's strictly testing behavior of the CLI without making any API calls, and the unit test that you added in this branch already covers that. If you do want to keep the test around, I recommend simplifying it as follows:

```
	t.Run("test named resource request with all-namespaces flag fails", func(t *testing.T) {
		_, err := TestHelper.ConduitRun("stat", "deploy/web", "--all-namespaces")
		if err == nil {
			t.Fatal("Expected to receive an error, got nothing")
		}
	})
```

There's no need to use `RetryFor`, since the func that you're passing never returns an error, and as such it will never be retried.

Reply:
yeah, I think I'll just remove this integration test. I think I originally wrote it in the wrong stat_test.go, moved it, and didn't remove this. Thanks!